libs/agno/agno/os/router.py

```diff
@@ -884,6 +980,8 @@ async def create_workflow_run(
         stream: bool = Form(True),
         session_id: Optional[str] = Form(None),
         user_id: Optional[str] = Form(None),
+        background: Optional[bool] = Form(True),
+        connection_id: Optional[str] = Form(None),
```

Comment:
I don't see the need for this... won't we just make a new connection for each run?

Reply:
discussed on slack